src/app/shared/issue-tables/issue-tables.component.html

```diff
@@ -106,11 +106,18 @@
         <mat-icon>link</mat-icon>
       </button>
       <button *ngIf="permissions.isTeamResponseEditable() && !issue.status
-       && this.isActionVisible(action_buttons.RESPOND_TO_ISSUE)"
+       && this.isActionVisible(action_buttons.RESPOND_TO_ISSUE); else tryFixIssue"
```

Comment:
I think there is an error in logic here. If the user do not have permission to edit the team then the user should also not be able to edit the issue. In that case, we should not be showing the icon.

Reply:
Thanks for the feedback!

Here is how I plan to fix this logic error:
- Create a `isIssueEditable(): boolean` method in the `PermissionsService`
- `isIssueEditable()` returns true if any of the permissions: `isIssueTitleEditable`, `isIssueDescriptionEditable`, `isIssueLabelsEditable`, `isTeamResponseEditable` or `isTutorResponseEditable` are true

- Call `permissions.isIssueEditable()` from the `issue-tables` component template and only display the edit icon if this returns true.

Would this be appropriate?

Reply:
LGTM